app/serializers/user_serializer.rb

```diff
@@ -2,7 +2,8 @@ class UserSerializer < ActiveModel::Serializer
   attributes :id, :cover_image_url, :avatar_template, :rating_type, :mini_bio,
     :about, :is_followed, :title_language_preference, :location, :website,
     :waifu, :waifu_or_husbando, :waifu_slug, :waifu_char_id, :last_sign_in_at,
-    :current_sign_in_at, :is_admin, :following_count, :follower_count, :is_pro
+    :current_sign_in_at, :is_admin, :following_count, :follower_count, :is_pro,
+    :to_follow
```

Comment:
Again, do we really need to include `to_follow`? If possible we should avoid it.